actions/q-162: ADD question r/t publshng Docker container actions (#726)
FidelusAleksander/ghcertified: questions/en/actions/question-162.md

---
question: "Mercedes wants to publish a Docker container action she has created to the GitHub Actions Marketplace. What files does she need at a minimum to do so?"
documentation: "https://docs.github.com/en/actions/how-tos/create-and-publish-actions/publish-in-github-marketplace"
---

- [x] `action.yml`
> An `action.yml` file is required for an action to be published to the Marketplace, regardless of type.

> - [x] A `Dockerfile`, if the image is located locally and must be built during the workflow run

Reviewer: If the `image` is located locally seems wrong.

A `Dockerfile` is not an image (it's essentially instructions on how to build an image) and I think thats what you were referring to

Author: A Dockerfile is what's used to build the image right? Should I reword that answer to say "image to be built"?

Reviewer: Maybe 

```suggestion
- [x] A `Dockerfile`, if the image is built as part of the action during the workflow run
```

Author: Committed. 
Sounds good to me!
